### ISSUE TITLE
Installed. What's next??

### ISSUE THREAD

**Issue by bolsoncerrado** [Salandora/octoprint-customControl#4]
Ok, Im feeling a total idiot right now. I've installed the plugin, restarted octoprint... I can see the custom controls on my CONTROL tab....but HOW do I edit them!?!?

Sorry!

**Comment by Salandora** [comment]
You need to open your settings and under plugins there is a caterogry Custom Controls.
There you need to right click either on one of your custom controls or inside the green box.

**Comment by bolsoncerrado** [comment]
we have an issue here then as I can't find a "custom control" option added
to the settings menu while the rest of plug-ins do :(

I'm on the last octoprint branch if that Helps?

**Comment by Salandora** [comment]
Can you check inside the plugin manager if the plugin is really installed.
also a clean octoprint.log could be helpful if the plugin is installed.
Otherwise (if plugin is not installed) the plugin manager log. I hope to find the readon why inside of one of this logs.

**Comment by bolsoncerrado** [comment]
<img width="966" alt="captura de pantalla 2016-07-01 a las 13 21 40" src="https://cloud.githubusercontent.com/assets/10992914/16520214/f44e0548-3f8e-11e6-8865-4f92efdb7db8.png">

**Comment by bolsoncerrado** [comment]
Its also weird that I click on GET MORE and i get the whole list of both installed and non installed plugins. Before, i used to get only the NON INSTALLED ones :??

**Comment by Salandora** [comment]
Ahh I see you have my Editor Collection plugin installed, so you should find the Custom Control Editor under Editor Collection, as well as the System Command Editor.

**Comment by bolsoncerrado** [comment]
I gave up on wheezy and now installing a fresh jessee... I'll report back
soon.


2016-07-01 17:40 GMT+02:00 Salandora <notifications@github.com>:

**Comment by Salandora** [comment]
Alright, I would recommend not to use the Editor Collection plugin at the moment, some users reported problems with it, but I'M lacking the time to inspect this bug.

**Comment by bolsoncerrado** [comment]
Yep on the fresh install of jessee I now get the CustomControl setting and
a green empty box for now.


2016-07-01 18:31 GMT+02:00 Salandora <notifications@github.com>:

**Comment by Salandora** [comment]
Awesome. I relly need to work on some documentation for this. But hell I dont have time and it is so nasty xD. 
Do you atill need help here otherwise I would close the issue.

**Comment by bolsoncerrado** [comment]
Im assuming I need to FIRST manually add SOMETHING to config.yaml otherwise
the plugin wont work properly, righT?


2016-07-01 20:26 GMT+02:00 Salandora <notifications@github.com>:

**Comment by Salandora** [comment]
Nope you just need to right click inside the greenbox.
There you can create a Container Element in which you can add your controls also with a right click.

**Comment by bolsoncerrado** [comment]
OK ill try and report back in the event it wouldnt work. Thanks!

**Comment by bolsoncerrado** [comment]
Managed to create several COMMANDS. How can I clone the new CONTAINER and COMMANDS to other raspi instances I have running in my network? or.... where are the changes being saved so I can overwrite the file(s) in the rest of raspis? :D

Thanks!

**Comment by Salandora** [comment]
You'll need to copy the config.yaml from Octoprint.
Or you better only copy the controls part out of it.

~/.octoprint/config.yaml

**Comment by bolsoncerrado** [comment]
cool thanks!

**Comment by amrithmmh** [comment]
can i write a raspberry command to control a GPIO pin and add it?
what is the commands that can be used in the custom commands?

**Comment by Salandora** [comment]
Custom Commands only allow commands send to the printer.
What you're looking for are the system commands, which can handle that.
